pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -5353,7 +5353,7 @@
   tutorial_careconfigure_name: "ReconFigure of Speech"
   tutorial_rpiscratch_name: "Scratch 3.0 phrasebook"
   tutorial_ozaria_name: "Ozaria by CodeCombat"
-  tutorial_ttovum_name: "Ovum City"
+  tutorial_ttovum_name: "Ovun City"
```

Comment:
Is it worth fixing the spelling in the keys as well?

Reply:
I don't think so.